### PR TITLE
show steps label

### PR DESCRIPTION
**Pull request by mahmutduva**
you can show steps label with this option

### CODE REVIEW

**Comment by ValentinH** [comment]
Thanks for this PR, this is a pretty cool feature! Just have a few remarks:

- this does not support using a translate function (if I want to add a '$' before each value for instance)
- this does not scale well (try replacing 600 by 60000 and 150 by 15000 in your example, then the labels are not positioned correctly)
- I don't really like the idea of modifying the `translateFn` like you did, I think this feature need its own function to position eveything (for example, like I did for the [show-ticks PR](https://github.com/rzajac/angularjs-slider/pull/68/files#diff-0539d03e9ce14a40acbd41c92f2caa2eR560) with the `updateTicksScale` function).

File: rzslider.js [hunk 1]
```diff
@@ -392,18 +412,34 @@ function throttle(func, wait, options) {
...
+
       useCustomTr = useCustomTr === undefined ? true : useCustomTr;
 
-      var valStr = String(useCustomTr ? this.customTrFn(value) : value),
-          getWidth = false;
+      if(label[0].className == 'rz-labels'){
+          var valStr = '';
+          console.log(this.barWidth)
+           for(var i=0; i<value.length;i++){
+                valStr += '<li style="left: '+ ((label.rzsw - 38) / (value.length - 1)) * i  +'px">' + value[i] + '</li>';
```
Comment (ValentinH) [inline rzslider.js]: What is 38?

**Comment by mahmutduva** [comment]
tick position bug fix
create new function 'updateTicksScale'

**Comment by ValentinH** [comment]
Indeed, this is much better like this.
Just to be perfect, check this example:
image

The labels are not centered... I don't know if there is a simple way to make them centered.

**Comment by ValentinH** [comment]
Moreover, the browser crashes if I don't provide the step argument!

**Comment by sebastialonso** [comment]
Any update on this PR?

**Comment by ValentinH** [comment]
Nope nothing since my last 2 remarks...

**Comment by mahmutduva** [comment]
I am working about label position. I will commit this update today.

**Comment by e-cloud** [comment]
@mahmutduva , can you build some simple demo to show the feature?

**Comment by ValentinH** [comment]
There you go: http://jsfiddle.net/aq2r7dL8/

**Comment by ValentinH** [comment]
@mahmutduva the labels still don't show if I don't set the step arg...

**Comment by ValentinH** [comment]
I'm closing this PR because I just add this feature on master with `rz-slider-show-ticks-value`.
See http://jsfiddle.net/0xg0bf2a/

**Pull request closed without merging**